machine-agent/templates/service.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
   ports:
     - port: {{ .Values.analytics.port }}
-      targetPort: http
+      targetPort: {{ .Values.analytics.port }}
```

Comment:
We'd rather prefer to have names based target ports as it is a recommended best practices
I can see why it's not working right now because "http" named port is not present in the daemonSet definition
If you can change this in daemonSet.yaml instead of service it'd make more sense
Thank you for the PR

Reply:
I'm sorry that I missed activity on this PR. It had sat for some time and I had been told that this particular chart was being moved away from, so I wasn't paying close attention.

Asking to correctly name and target the port itself makes sense. An update doing this has been committed.